types/node-gamepad/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "noImplicitAny": true,
```

Comment:
tslint should contain only extends clause. Nothing else.

Reply:
Got it, the checklist item from the PR template made me think I might have to also manually add these items.